src/storage-preview/azext_storage_preview/_params.py

```diff
@@ -161,3 +163,34 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('error_document_404_path', options_list=['--404-document'], arg_group='Static Website',
                    help='Represents the path to the error document that should be shown when an error 404 is issued,'
                         ' in other words, when a browser requests a page that does not exist.')
+
+    with self.argument_context('storage azcopy blob upload') as c:
+        c.extra('destination_container', options_list=['--container', '-c'], required=True)
+        c.extra('destination_path', options_list=['--destination', '-d'],
+                validator=validate_azcopy_upload_destination_url)
+        c.argument('source', options_list=['--source', '-s'])
+        c.argument('recursive', options_list=['--recursive', '-r'], action='store_true')
+        c.ignore('destination')
+
+    with self.argument_context('storage azcopy blob download') as c:
+        c.extra('source_container', options_list=['--container', '-c'], required=True)
+        c.extra('source_path', options_list=['--source', '-s'],
+                validator=validate_azcopy_download_source_url)
+        c.argument('destination', options_list=['--destination', '-d'])
+        c.argument('recursive', options_list=['--recursive', '-r'], action='store_true')
+        c.ignore('source')
+
+    with self.argument_context('storage azcopy blob delete') as c:
+        c.extra('target_container', options_list=['--container', '-c'], required=True)
+        c.extra('target_path', options_list=['--target', '-t'],
+                validator=validate_azcopy_target_url)
+        c.argument('recursive', options_list=['--recursive', '-r'], action='store_true')
+        c.ignore('target')
+
+    # with self.argument_context('storage azcopy blob sync') as c:
+    #     c.argument('destination', options_list=['--destination', '-d'],
+    #                validator=validate_azcopy_container_destination_url)
+    #     c.argument('source', options_list=['--source', '-s'])
```

Comment:
Why these are commented out? Not ready or not working?

Reply:
Because of: https://github.com/Azure/azure-storage-azcopy/issues/255. When azcopy releases again, we can re-test and add the functionality if it is in a good state.